IBPSA/Obsolete/Fluid/FixedResistances/BaseClasses/Validation.mo

```diff
@@ -0,0 +1,153 @@
+within IBPSA.Obsolete.Fluid.FixedResistances.BaseClasses;
+package Validation "Collection of validation models"
+  extends Modelica.Icons.ExamplesPackage;
+
+  model PlugFlowCore "Simple example of plug flow pipe core"
```

Comment:
It would be better to split this package in multiple .mo files?

Reply:
Good catch! Updated